test-files/io/GeoJson/MultiObjectsReader_all.geojson

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
This file is a placeholder for future tests.  I get errors in the MultiPoint and MultiLine when I try to run it.
I think that either my GeoJSON is dorked up or the parser isn't reading it right. The error I get is the resulting relation is missing a way.